.github/actions/next-stats-action/Dockerfile

```diff
@@ -4,24 +4,25 @@ LABEL com.github.actions.name="Next.js PR Stats"
 LABEL com.github.actions.description="Compares stats of a PR with the main branch"
 LABEL repository="https://github.com/vercel/next-stats-action"
 
-COPY . /next-stats
-
 RUN apt update && apt upgrade -y
-RUN apt install unzip wget curl nano htop screen build-essential pkg-config libssl-dev git build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libreadline-dev libffi-dev python3 moreutils jq iproute2 openssh-server sudo whois dnsutils -y
+RUN apt install unzip wget curl nano htop screen build-essential pkg-config libssl-dev git build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libreadline-dev libffi-dev python3 moreutils jq iproute2 openssh-server sudo whois dnsutils apache2-utils -y
 
 RUN ln $(which python3) /usr/bin/python
 
 RUN curl -sfLS https://install-node.vercel.app/v18.16.1 | bash -s -- -f
+RUN corepack enable
+
+WORKDIR /next-stats
 
 # Install node_modules
-RUN npm i -g pnpm@7.24.3 yarn@1.22.19
```

Comment:
We need to keep the exact version of pnpm and yarn here

Reply:
Why? `pnpm` is exact from corepack and I don't know where yarn is used, but it should probably use corepack as well? 🤔

Reply:
Unfortunately, corepack doesn't support multiple package managers per project. 

So perhaps any tests that rely on yarn will need to add `"packageManager": "yarn@1.22.19"`